### PR TITLE
add Selects multiple values

### PR DESCRIPTION
**Pull request by LiKang6688**
works on #8

### CODE REVIEW

File: src/table/selections-utils.js [hunk 1]
```diff
@@ -80,3 +80,36 @@ export function selectCell(cell, selState, selDispatch, evt) {
     api.cancel();
   }
 }
+
+export const doSelectCell = (evt, rowValues, rowIndex, colId, selState, selDispatch) => {
```
Comment (LiKang6688) [inline src/table/selections-utils.js:84]: not sure it is the correct way.  If yes, I will refactor it.

Comment (haxxmaxx) [reply src/table/selections-utils.js:84]: Is this for selecting multiple?

Comment (haxxmaxx) [reply src/table/selections-utils.js:84]: Ah, nevermind. This is a new PR...

Comment (LiKang6688) [reply src/table/selections-utils.js:84]: Yes, 
```
 if (keysPressed.Shift && (evt.key === 'ArrowUp' || evt.key === 'ArrowDown')) {
     doSelectCell(evt, rows, rowIndex, colId, selState, selDispatch);
 }
```

**Comment by haxxmaxx** [comment]
This branch could be saved and be used if we implement this later, but the PR can be closed

**Pull request closed without merging**